testsuite/features/support/http_client.rb

```diff
@@ -13,7 +13,9 @@ def initialize(host)
   def prepare_call(name, params)
     short_name = name.split('.')[-1]
     call_type =
-      if short_name.start_with?('list', 'get', 'is', 'find') || ['logout', 'errata.applicableToChannels'].include?(name)
+      if short_name.start_with?('list', 'get', 'is', 'find') ||
+         name.start_with?('system.search.', 'packages.search.') ||
+         ['auth.logout', 'errata.applicableToChannels'].include?(name)
```

Comment:
I would prefer a unique list in include to treat all those:


```suggestion
         api_methods = ['auth.logout', 'errata.applicableToChannels', 'system.search.', 'packages.search.']
         api_methods.include?(name)
```

Reply:
won't work, we need `system.search.hostname`, `system.search.ip`, etc. `include?` does an exact match.

Reply:
Good idea. We already have an endpoint to list all the namespaces and methods. We should add the GET/POST info to it.

Then if you want to invest, you can request the list at the beginning of each run. That way you won't have to update this list when something changes.

Reply:
Check out the methods in the 'api' namespace